### PR TITLE
utility to generate dummy data

### PR DESCRIPTION
**Pull request by asakapab0i**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix
- [x] New feature

### Testing

- [ ] Testing information has been added - test cases checklist and steps followed for testing.
- [ ] Testing screenshot(s) attached for more information.

## Security

- [ ] This PR has security related considerations.

### CODE REVIEW

**Review by ttheune — CHANGES_REQUESTED** [review]
Looks good.

**Review by ttheune — APPROVED** [review]
looks good

**Comment by danishfoxpass** [comment]
@asakapab0i i have updated the PR with required changes. You can test and merge.

**Review by danishfoxpass — APPROVED** [review]

File: api/foxpass_dummy_data.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            log('Successfully added custom fields {} for the user {}'.format(cf_list, user))
+        else:
+            log('Error adding custom fields {} for the user {}. Pls add custom user fields here first: {}'.format(cf_list, user, CONSOLE_PAGE_URL))
+
+
+# Generate values based on the cf type
+def generate_value_cf_type(cf, fake):
+    try:
+        # following will call a function named cf if that exists for faker and it's modules
+        return str(getattr(fake, cf)())
```
Comment (asakapab0i) [inline api/foxpass_dummy_data.py:132]: This is cool, I've looking for something like this.

**Review by asakapab0i — COMMENTED** [review]
Looks good, tested.

**Pull request merged**